pillow 10.3.0 prohibits daily build

```
webviz_subsurface/plugins/_map_viewer_fmu/_utils.py:21: error: Module "PIL.Image" is not valid as a type  [valid-type]
webviz_subsurface/plugins/_map_viewer_fmu/_utils.py:21: note: Perhaps you meant to use a protocol matching the module structure?
webviz_subsurface/plugins/_map_viewer_fmu/_utils.py:24: error: Image? has no attribute "save"  [attr-defined]
Found 2 errors in 1 file (checked 532 source files)
```

This seems to have been introduced around 2 April, so the project has not built for almost a month now.
https://github.com/equinor/webviz-subsurface/actions/runs/8515475794

The previous day it worked;
https://github.com/equinor/webviz-subsurface/actions/runs/8501941722/job/23285458791

https://pypi.org/project/pillow/10.3.0/#history

--- a/webviz_subsurface/plugins/_map_viewer_fmu/_utils.py
+++ b/webviz_subsurface/plugins/_map_viewer_fmu/_utils.py
@@ -18,10 +18,10 @@ def rgb_to_hex(color: List[int]) -> str:
     return f"#{color[1]:02x}{color[2]:02x}{color[3]:02x}"
 
 
-def image_to_base64(img: Image) -> str:
+def image_to_base64(img: Image) -> str:  # type: ignore[valid-type]
     """Convert an image to a base64 string."""
     buffer = io.BytesIO()
-    img.save(buffer, format="PNG")
+    img.save(buffer, format="PNG")  # type: ignore[attr-defined]
     buffer.seek(0)
     return base64.b64encode(buffer.read()).decode("utf-8")
 

